docs/authentication/social-connections/apple.mdx

```diff
@@ -17,115 +17,111 @@ description: Learn how to allow users to sign into your Clerk app with their App
+  1. Select **Next**. You'll be redirected to the **Confirm your web authentication configuration** screen.
+  1. Select **Done**. You'll be redirected to the **Edit your Services ID Configuration** page.
+  1. Select **Continue**. You'll be redirected to the confirmation page.
+  1. Select **Save**. You'll be redirected to the **Identifiers** page.
 
   ### Get your Apple Private Key and Key ID
 
-  The **Apple Private Key** and **Key ID** are _required_ for web based OAuth flows and _recommended_ for native app flows.
+  The **Apple Private Key** and **Key ID** are _required_ for web-based flows and _recommended_ for native app flows.
 
-  To get your **Apple Private Key** and **Key ID**, you must create a new **Key** in the Apple Developer portal.
+  To get your **Apple Private Key** and **Key ID**, create a new **Key** in the Apple Developer portal.
 
-  1. You should be back at the **Identifiers** page.
-  1. In the sidebar, select **Keys**.
-  1. Next to **Keys** at the top of the page, select the plus icon (+) to register a new key.
-  1. On the **Register a New Key** page, add a **Key Name** and ensure the **Sign In with Apple** box is enabled and select **Configure**. You'll be redirected to the **Configure Key** page.
-  1. Under **Primary App ID**, select the **App ID** you created in the first step of this guide. Then select **Save**. You'll be taken back to the previous **Register a New Key** page.
-  1. Select **Continue** and you'll be presented with the final confirmation screen. Verify that **Sign in with Apple** is checked and select **Register**. You'll be redirected to the **Download Your Key** page.
-  1. Save the **Key ID** value as you'll need to supply it to Clerk later.
-  1. Download the private key as a file (as the instructions point out, be sure to backup the key in a secure place as it cannot be redownloaded later).
-  1. Select **Done**.
+  1. On the **Identifiers** page, in the sidebar, select **Keys**.
+  1. Next to **Keys** at the top of the page, select the plus icon (+) to register a new key. You'll be redirected to the **Register a New Key** page.
+  1. Add a **Key Name** and ensure the **Sign In with Apple** box is enabled and select **Configure**. You'll be redirected to the **Configure Key** page.
+  1. Under **Primary App ID**, select the **App ID** you created in the first step of this guide. Then select **Save**. You'll be redirected to the previous **Register a New Key** page.
+  1. Select **Continue** and you'll be presented with the final confirmation screen. Verify that **Sign in with Apple** is checked. Select **Register**. You'll be redirected to the **Download Your Key** page.
+  1. Save the **Key ID** value somewhere secure. This is your **Apple Key ID** in Clerk.
+  1. Download the private key as a file. This is your **Apple Private Key** in Clerk. As the instructions point out, be sure to backup the key in a secure place as it cannot be redownloaded later.
```

Comment:
```suggestion
  1. Download the private key file. This will serve as your **Apple Private Key** in Clerk. Ensure you back up the key in a secure location, as it cannot be downloaded again later.
```